helmfile.d/helmfile-09.init.yaml

```diff
@@ -24,12 +24,12 @@ releases:
       pkg: knative
     values:
       - ../values/knative/knative-serving-raw.gotmpl
-  - name: gatekeeper-operator-config
+  - name: gatekeeper-operator-artifacts
```

Comment:
rename chart `gatekeeper-operator` to `gatekeeper` and this one to `gatekeeper-artifacts`